README.rst

```diff
@@ -211,6 +211,49 @@ in that folder.
 
     mkdir -m /var/log/fiware-cloto
 
+Running with supervisor
+-----------------------
+
+Optionally you can add a new layer to manage gunicorn proccess with a supervisor.
+Just install supervisor on your system:
+
+::
+
+    sudo apt-get install supervisor
+
+Copy the file ``utils/cloto_start`` to ``/etc/fiware.d``.
```

Comment:
I do not know if `/etc` is a good place for executable files... What do you think about `/opt/fiware/fiware-cloto/cloto_start` as default value?